apps/docs/content/guides/platform/regions.mdx

```diff
@@ -1,18 +1,19 @@
 ---
 title: Available regions
-subtitle: Spin up Supabase projects in our global regions
```

Comment:
We don’t seem to be using subtitles in other docs nearby. Figured it was redundant.